deployment/testing-v3/compose/admin/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.4"
+
+services:
+  app:
+    image: mozilla/prio-processor:v3.1.1
```

Comment:
I've pinned the containers here for the sake of reproducibility. I'm going to break a lot of the current conventions in order to support #112